.github/ISSUE_TEMPLATE/offboarding-request.md

```diff
@@ -1,5 +1,5 @@
 ---
-name: Offboarding request
+name: Individual offboarding request
```

Comment:
Can we change this to offboarding request (individual) per my comment under Conversation?